scripts/update.js

```diff
@@ -213,20 +213,17 @@ const apply = (change) => {
   const curr = packages[name]
 
   if (change.deleted) {
-    if (typeof curr === 'string') {
-      updateRepoStats(curr, -1)
-    }
+    updateRepoStats(curr, -1)
 
     stats.deletes += 1
     return delete packages[name]
   }
 
-  const url = extractUrl(change)
-
-  if (!url) {
-    stats.invalid += 1
-    return
-  }
```

Comment:
It was nice to have that stat in the metadata, can we keep it? it would be useful for, in ie, updating the readme.

Reply:
Yeah, I was thinking about it. We can calculate "invalid" packages based on `packages.json` since we keep all of them now. But the idea to have it pre-calculated in metadata is probably better.

Reply:
btw, what packages should be counted as invalid now? If we add `repos.unset` statistics it will have exactly the same numbers as `stats.invalid`